npm_and_yarn/lib/dependabot/npm_and_yarn/helpers.rb

```diff
@@ -16,6 +16,14 @@ def self.npm_version_numeric(lockfile_content)
         6
       end
 
+      def self.yarn_version_numeric(lockfile_content)
+        if yarn_berry?(lockfile_content)
+          3
+        else
+          1
+        end
```

Comment:
🔎 Can we use [yarn_major_version](https://github.com/dependabot/dependabot-core/pull/7087/files#diff-85ec3865f2fdb48045a2536b37b847156ed3bd3960afdc8bd900ae568de9acd7R42) for this?

Reply:
This is tricky stuff. `yarn --version` would give us the version Dependabot is running (which will always be Yarn Berry I believe). I think it's more interesting to know the version the project needs, and the `yarn_berry?` helper is the criteria used in the update checker and other places to decide whether to run Yarn Berry, or Yarn 1 native helpers. So, I think `yarn_berry?` is more reliable/informative actually.